drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -379,6 +385,12 @@ local function device_type_supports_button_switch_combination(device, endpoint_i
     if ep.endpoint_id == endpoint_id then
       for _, dt in ipairs(ep.device_types) do
         if dt.device_type_id == DIMMABLE_LIGHT_DEVICE_TYPE_ID then
+          for _, fingerprint in ipairs(child_device_profile_overrides_per_vendor_id[0x115F]) do
+            if device.manufacturer_info.product_id == fingerprint.product_id then
+              -- For Aqara Dimmer Switch with Button.
+              return false
```

Comment:
nit since I left a couple other comments:
```suggestion
              return false -- For Aqara Dimmer Switch with Button.
```

Reply:
Thanks, I applied it.